### DISCUSSION TITLE
Google ad on home page?!

### DISCUSSION THREAD

**Discussion by dotku** [hexojs/hexo#5375] [Question / Answer] [answered]
Not sure if I'm been hacked, I saw current document site full of google AD. Is this project facing any financial difficulty; I wonder how much google ad can help Hexo.

**Comment by SukkaW** [comment]
This is probably not caused by Hexo. The Google AD is most likely introduced by the theme you used.

**Reply by dotku** [reply]
Did you see something like

![image](https://github.com/hexojs/hexo/assets/1519232/db2aa43e-ea2c-4c2c-a68e-3d132c6503e6)

**Reply by yoshinorin** [reply, marked as answer]
@dotku cc: @SukkaW @uiolee 

IMHO, we should remove google ads from our site.

### Reasons

* The current active maintainers can't manage it (we don't have a permission) 
* I assume google ads's revenue is very few and it not helpful us
* unfriendly to users

-----

I'll submit a PR.

Thank you :)

-----

Submitted: https://github.com/hexojs/site/pull/2121

**Reply by D-Sketon** [reply]
should carbonads also be removed?
https://github.com/hexojs/site/blob/master/themes/navy/layout/partial/carbonads.njk

**Reply by yoshinorin** [reply]
@D-Sketon 

> should carbonads also be removed?

Yes 👍 Would you please submit a PR?

**Comment by uiolee** [comment]
From 2018, Google ads is inserted into `hexo.io`. https://github.com/hexojs/site/commit/6e5d82bc9cf59664704cb36ccb38e48e0998fd98

previously, ~In my view, I can only see the bottom ads.~

There are more ads now.

![图片](https://github.com/hexojs/hexo/assets/22849383/990c52fa-11d7-40df-8b88-cb06216df472)

**Comment by dotku** [comment]
Yes, could offer a sponsor options, I believe many tech companies would like to prompt their product, could use eg petron to collect fund. What about AI project to use Nexojs to create the website, I think it would be cool project to work on :)

Also could refer to Tailwind CSS provide some professional service/business templates
